python/pyhdk/_storage.pxd

```diff
@@ -131,7 +131,7 @@ cdef extern from "omniscidb/ArrowStorage/ArrowStorage.h":
     CArrowStorage(int, string, int, shared_ptr[CConfig]) except +;
 
     CTableInfoPtr createTable(const string&, const vector[CColumnDescription]&, const CTableOptions&) except +
-
+    CTableInfoPtr createRefragmentedView(const string& , const string&, const size_t) except +
```

Comment:
Our main API access point is `pyhdk.HDK` class. Do you use it in your demo? I'd expect usage scenario to be something like:
```
ht1 = hdk.import_csv(file, fragment_size=10)
...
ht2 = ht1.refragmented_view(fragment_size=20)
...
```

Reply:
Demo will be redone as a separate PR.